label_sleuth/app.py

```diff
@@ -535,17 +535,16 @@ def query(workspace_id):
     query_string = request.args.get('qry_string')
```

Comment:
Can you also modify the docstring of the `app.query()` endpoint?

Reply:
done, please verify, it said `regular expression`, I changed it to just `query string`